config/environments/development.rb

```diff
@@ -58,4 +63,16 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :smtp
+  ActionMailer::Base.smtp_settings = {
+    :address        => 'smtp.gmail.com',
+    :port           => '587',
+    :authentication => :plain,
+    :user_name      => ENV['gmail_username'],
```

Comment:
em đổi cả file này từ nháy đơn thành nháy đôi nhé, và dùng biến in hoa cho mấy cái này
VD: GMAIL_USERNAME